storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc

```diff
@@ -16,6 +16,13 @@ include::modules/secrets-store-providers.adoc[leveloffset=+2]
 
 include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
+include::modules/persistent-storage-csi-secrets-store-disconnect-environment.adoc[leveloffset=+1]
```

Comment:
I know you didn't create this assembly, but can you please add [this metadata](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc) to the top of the file? I had trouble finding the topic in the preview. 

`:_mod-docs-content-type: ASSEMBLY`